agent-lab/docs/environment-setup-local-virtualenv.md

```diff
@@ -10,19 +10,16 @@ Please ensure your development environment has the following:
 - [Python](https://www.python.org/downloads/) (versions 3.10, 3.11 and 3.12)
 - [Poetry](https://python-poetry.org/) for Python dependency management.
 - The ability to run Make commands
```

Comment:
nit: put a reference?
```suggestion
- The ability to run [Make](https://www.gnu.org/software/make/manual/make.html) commands
```

Reply:
@LauraDamianTNA applied. Can we merge this PR?